Demo/Application/Features/PayPal - Native Checkout/BraintreeDemoPayPalNativeCheckoutViewController.swift

```diff
@@ -5,36 +5,46 @@ import BraintreePayPalNativeCheckout
 class BraintreeDemoPayPalNativeCheckoutViewController: BraintreeDemoPaymentButtonBaseViewController {
     lazy var payPalNativeCheckoutClient = BTPayPalNativeCheckoutClient(apiClient: apiClient)
 
+    func checkoutPaymentButton(title: String) -> UIButton {
```

Comment:
It looks like this method is set to 2 spaces whereas our SDK uses 4 - I know this is different than NXO but we will want to update this to 4 spaces.